### ISSUE TITLE
Unknown X server provider (AMD GPU)

### ISSUE THREAD

**Issue by FFY00** [ashwinvis/xrandr-extend#7]
* xrandr-extend version: 0.2.0
* Python version: 3.7.4
* Operating System: Archlinux

### Description
xrandr-extend can't identify my provider.

### What I Did

```
$ xrandr-extend --dry-run anything-here
Unknown X server provider. Output of `xrandr --listproviders` was: 00.0
Hint: perhaps you are using Wayland and not an X server? If not simply add a section [provider:00.0] in /home/anubis/.config/xrandr-extend.cfg
$ xrandr --listproviders
Providers: number : 1
Provider 0: id: 0x59 cap: 0xf, Source Output, Sink Output, Source Offload, Sink Offload crtcs: 6 outputs: 4 associated providers: 0 name:Unknown AMD Radeon GPU @ pci:0000:03:00.0
```

**Comment by ashwinvis** [comment]
Hmm... I did not expect the profiders to be inconsistent: For my laptop with Intel graphics I get:

```sh
❯ xrandr --listproviders
Providers: number : 1
Provider 0: id: 0x45 cap: 0xf, Source Output, Sink Output, Source Offload, Sink Offload crtcs: 3 outputs: 3 associated providers: 0 name:modesetting
```

We need a better regex to parse that. For now, as the hint suggests modify `~/.config/xrandr-extend.cfg` to add a section as:

```cfg
[provider:00.0]
# insert monitors here
```

for monitors try:

```sh
❯ xrandr --listmonitors
Monitors: 1
 0: +*eDP-1 3200/294x1800/165+0+0  eDP-1
```

**Comment by ashwinvis** [comment]
Should be fixed in #8. You should be able to do:
```cfg
[provider:Unknown AMD Radeon GPU @ pci:0000:03:00.0]
# insert monitors here
```
